Add travis for unit tests

mareklibra opened the pull request:
So far integration tests are executed only.
Integration tests would require running OpenShift.

Fixes: https://github.com/kubevirt/web-ui/issues/4

Reopen #10

mareklibra commented:
So disabling all tests except one helps. Looks like issue with the tests.
Needs more investigation.

ohadlevy commented:
would it be easier to use xit (instead of it) to skip the "bad" tests instead of moving files around?

mareklibra commented:
Fixed - `jest` is not able to properly read number of CPUs in travis environment (32 instead of 2). Maybe "CPU hotplug"-effect, but it's not important for us now.

Together with `ts-jest` preprocessor, this is a killing combination.

Fixed by limiting number of workers to 2, works smoothly now.
